app/templates/_package.json

```diff
@@ -19,7 +19,7 @@
     "grunt-contrib-connect": "~0.5.0",
     "connect-livereload": "~0.3.0",
     "grunt-grunticon": "~1.0.0",
-    "grunt-bower": "~0.7.0",
+    "grunt-bower": "^0.18.0",
```

Comment:
Do we want to be so lenient with version matching?  `^` matches any 0.* version while `~` matches 0.18.*.